x-pack/plugins/observability_solution/logs_explorer/public/customizations/logs_explorer_profile.tsx

```diff
@@ -130,7 +131,19 @@ export const createLogsExplorerProfileCustomizations =
         },
       },
       docViewsRegistry: (registry) => {
-        registry.enableById('doc_view_logs_overview');
+        registry.add({
+          id: 'doc_view_logs_overview',
+          title: i18n.translate('xpack.logsExplorer.docViews.logsOverview.title', {
+            defaultMessage: 'Overview',
+          }),
+          order: 0,
+          component: (props) => (
+            <UnifiedDocViewerLogsOverview
+              {...props}
+              renderAIAssistant={plugins.logsShared.renderLogsAIAssistant}
```

Comment:
> [!NOTE]
> What if, instead of exporting a new `logsShared.renderLogsAIAssistant` renderer and introducing the dependency on `logsShared`, we directly consume this from the `discoverShared` registered feature as it was doing directly in the UnifiedDocViewer? It should be fine to introduce the dependency on discover-shared and consume the registered feature, also avoiding to update the signature of the logsShared plugin at all, wdyt?
```ts
const logsAIAssistantFeature = discoverShared.features.registry.getById(
  'observability-logs-ai-assistant'
);
registry.add({
  id: 'doc_view_logs_overview',
  title: i18n.translate('xpack.logsExplorer.docViews.logsOverview.title', {
    defaultMessage: 'Overview',
  }),
  order: 0,
  component: (props) => (
    <UnifiedDocViewerLogsOverview
      {...props}
      renderAIAssistant={logsAIAssistantFeature.render}
```

Reply:
Good suggestion, thanks! I like that better and I suppose that's why we introduced `discoverShared` in the first place 🙂 Updated here: https://github.com/elastic/kibana/pull/186680/commits/97ba3c1e31c3accd5ed57ec326e3e0dda5440368.